### DISCUSSION TITLE
loose bufferline bar if switch session that not current in a session?

### DISCUSSION THREAD

**Discussion by tmpm697** [rmagatti/auto-session#197] [General]
more in here: https://github.com/rmagatti/auto-session/issues/196#issuecomment-1377064954

tools:
```
https://github.com/nvim-telescope/telescope-project.nvim
https://github.com/rmagatti/auto-session
```

```
I got this issue, step (clean up/remove all saved session files from cache):

1. use :Telescope project to create project/session A and B (with different directory)
2. cd "$(mktemp -d)"
3. nvim
4. :Telescope project and then ctrl-l to switch session A, and then open some buffers
5. same as 4 but with B
6. switch back and forth between A and B --> perfect, bufferline.nvim show its bar and buffers's name/title
7. exit nvim
8. nvim
9. select project A and B back and forth 
10. observe that there're no bufferline's bar exist --> bufferline completely not appear

why and how bufferline disappear when restore session and its' not happen in first session when create them. @rmagatti 

after 10 if i go to A project's folder and then start nvim, bufferline shows as usual.
```

sum up: 
1. if first open nvim and create projects and switch between them, you don't need to under a session, still work perfect switching
2. after projects created, you NEED to under a session/project to have bufferline bar appear between switching, if not you'll loose bufferline bar.

**Comment by rmagatti** [comment]
Yeah so like I mentioned on the issue. This doesn't seem to have anything to do with auto-session. I for one don't even use a bufferline plugin, so it'd be hard to help you there too. Filing an issue or starting a discussion on whatever bufferline plugin you use might be a way to go, also adding videos/gifs showcasing your issue clearly is generally a good addition as well. Good luck!

**Reply by tmpm697** [reply]
Thank you, this issue currently limit my workflow a bit as i have to cd to one of project to be able to switch project/session. I'll fire this on bufferline also and see if they can help.

**Reply by tmpm697** [reply]
switched to https://github.com/noib3/nvim-cokeline and don't see this issue: https://github.com/akinsho/bufferline.nvim/issues/655
